queue/src/main/java/org/killbill/queue/DBBackedQueue.java

```diff
@@ -756,4 +756,14 @@ public Long execute() throws RuntimeException {
             }
         });
     }
+
+    public void updateCreatingOwner(final Date reapingDate) {
+        sqlDao.inTransaction(new Transaction<Void, QueueSqlDao<T>>() {
```

Comment:
One more comment: it would be good to add some logging, probably `WARN` level, in case we do update some rows -- this should be uncommon.